features/security/src/main/feature/feature.xml

```diff
@@ -21,694 +21,317 @@
+        <bundle start-level="40">mvn:org.apache.cxf/cxf-rt-rs-security-jose/${cxf.version}</bundle>
 
-    <feature name="ldap-embedded-default-stslogin-config" version="${project.version}"
-             description="Installs a default embedded ldap claims handler config">
-        <configfile
-                finalname="${ddf.etc}/Ldap_Login_Config-ddf.ldap.ldaplogin.LdapLoginConfig.config"
-                override="false">
-            mvn:ddf.security.jaas/security-jaas-ldap/${project.version}/config/default-ldaplogin
-        </configfile>
-    </feature>
+        <!-- Pax -->
+        <config name="org.ops4j.pax.web" append="true">
+            #######################################
+            # Jetty Server Authenticator Settings
+            #######################################
 
-    <feature name="security-handler-basic" version="${project.version}"
-             description="Basic Authentication Handler for web applications.">
-        <feature>security-core-misc</feature>
-        <feature>parser-xml</feature>
-        <feature>security-core-services</feature>
-        <bundle>mvn:ddf.security.handler/security-handler-basic/${project.version}</bundle>
-    </feature>
+            # Set the authmethod/realmname so that our JettyAuthenticator is used
+            org.ops4j.pax.web.default.authmethod=DDF
+            org.ops4j.pax.web.default.realmname=DDF
+        </config>
+        <bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-jsp/${pax.web.version}</bundle>
 
-    <feature name="oidc-dependencies" version="${project.version}"
-      description="All OIDC third party dependencies">
-        <feature prerequisite="true">wrap</feature>
-        <feature>jackson</feature>
-        <bundle>mvn:com.sun.mail/javax.mail/${javax-mail.version}</bundle>
-        <!--
-            net.minidev/accessors-smart/1.2 uses asm v5.x. Need this resolved before we can upgrade:
-            https://bitbucket.org/connect2id/nimbus-jose-jwt/issues/239/switch-to-other-json-library
-        -->
-        <bundle>mvn:org.ow2.asm/asm/5.2</bundle>
-        <bundle>mvn:net.minidev/accessors-smart/1.2</bundle>
-        <bundle>mvn:net.minidev/json-smart/2.3</bundle>
-        <bundle>wrap:mvn:com.nimbusds/nimbus-jose-jwt/${nimbus.version}</bundle>
-        <bundle>wrap:mvn:com.nimbusds/oauth2-oidc-sdk/${nimbus.oidc.version}</bundle>
-        <bundle>mvn:com.nimbusds/content-type/${nimbus.content-type.version}</bundle>
-        <bundle>mvn:com.nimbusds/lang-tag/${nimbus.langtag.version}</bundle>
-        <bundle>wrap:mvn:com.github.stephenc.jcip/jcip-annotations/1.0-1</bundle>
-        <bundle>mvn:commons-codec/commons-codec/${commons-codec.version}</bundle>
-        <bundle>mvn:ddf.platform.security/security-oidc-bundle/${project.version}</bundle>
+        <!-- Miscellaneous -->
+        <bundle>mvn:com.google.code.gson/gson/${gson.version}</bundle>
+        <bundle>wrap:mvn:org.springframework.ldap/spring-ldap-core/2.3.2.RELEASE</bundle>
```

Comment:
Should all ldap functionality be contained in the ldap-auth feature? (and if this isn't used for ldap auth, I'm curious what it is used for)